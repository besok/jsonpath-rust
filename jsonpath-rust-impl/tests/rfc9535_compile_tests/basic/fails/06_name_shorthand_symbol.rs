//! Test case: 06_name_shorthand_symbol
//! Tags: No tags

use ::jsonpath_ast::ast::Main;
use ::jsonpath_rust_impl::json_query;

fn main() {
    assert_eq!(
        json_query! ( $.& ),
        Main::try_from_pest_parse(r#"$.&"#).expect("failed to parse")
    );
}
