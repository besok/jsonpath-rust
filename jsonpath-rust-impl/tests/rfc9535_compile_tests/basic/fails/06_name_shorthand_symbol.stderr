error: unexpected token
 --> tests/rfc9535_compile_tests/basic/fails/06_name_shorthand_symbol.rs:9:25
  |
9 |         json_query! ( $.& ),
  |                         ^
